core/services/keeper/validate.go

```diff
@@ -79,9 +63,5 @@ func ValidatedKeeperSpec(tomlString string) (job.Job, error) {
 		return j, errors.Errorf("unsupported type %s", j.Type)
 	}
 
-	if !reflect.DeepEqual(j.Pipeline.Tasks, expectedPipeline.Tasks) {
-		return j, errors.New("invalid observation source provided")
-	}
-
```

Comment:
Can we expect an empty observation source here? Since we'll be effectively ignoring what the user specifies, it will be safer if we make it an expectation that user needs to provide an empty observation source. It can be our final migration of observation sources

Reply:
Wouldn't it be better if the user just doesn't pass an observation source at all rather just passing an empty string? I would be happy to check if it is empty too, just wondering.